### PR TITLE
code challenge 8 done

### PR DESCRIPTION
**Pull request by daesy13**

### CODE REVIEW

**Comment by daesy13** [comment]
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

File: code401challenges/src/test/java/code401challenges/linkedList/LinkedListTest.java [hunk 1]
```diff
@@ -73,4 +73,49 @@
...
+
+        LinkedList actual2 = new LinkedList();
+        actual2.insert(5);
+        actual2.insert(9);
+        actual2.insert(4);
+
+        LinkedList actual = LinkedList.mergeLists(actual1, actual2);
+        String expected = "{3} -> {4} -> {2} -> {9} -> {1} -> {5} -> NULL";
+        assertEquals(expected, actual.toString());
+    }
```
Comment (jeffborda) [inline code401challenges/src/test/java/code401challenges/linkedList/LinkedListTest.java]: These tests are ok, but really try to avoid testing another method like `merge`using the `toString()` method. The problem is that if the format of `toString` gets changed down the line, then all these tests will fail even though there is nothing wrong with them.

File: code401challenges/src/test/java/code401challenges/linkedList/LinkedListTest.java [hunk 2]
```diff
@@ -73,4 +73,49 @@
...
+        actual1.insert(5);
+        actual1.insert(9);
+        actual1.insert(4);
+
+        LinkedList actual2 = new LinkedList();
+
+        LinkedList actual = LinkedList.mergeLists(actual1, actual2);
+        String expected = "{4} -> {9} -> {5} -> NULL";
+        assertEquals(expected, actual.toString());
+    }
```
Comment (jeffborda) [inline code401challenges/src/test/java/code401challenges/linkedList/LinkedListTest.java]: This is good too since one list is empty. I like that you are thinking of different edge cases like this.
